tests/scripts/all.sh

```diff
@@ -3217,6 +3209,8 @@ component_test_tfm_config_p256m_driver_accel_ec () {
     not grep mbedtls_rsa_ library/rsa.o
     not grep mbedtls_dhm_ library/dhm.o
     not grep mbedtls_mpi_ library/bignum.o
+    # Check that p256m was built
+    grep -q p256_ecdsa_verify library/libmbedcrypto.a
```

Comment:
I'm not 100% sure, but I seem to remember that I tried this and it didn't work - so I had to use `p256_ecdsa_verify`. So I'll leave this, unless you've tested it and found otherwise?